common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1601,3 +1601,54 @@ def can_toggle_course_pacing(self):
           bool: False if the course has already started, True otherwise.
         """
         return datetime.now(UTC()) <= self.start
+
+
+class CourseSummary(object):
+    """
+    A lightweight course summary class, which constructs split/mongo course summary without loading
+    the course. It is used at cms for listing courses to global staff user.
+    """
+    default_display_name = "Empty"
+    course_info_fields = ['display_name', 'display_coursenumber', 'display_organization']
+
+    def __init__(self, course_locator, course_metadata):
```

Comment:
Rather than take a dict (or `**kwargs`) of arguments, you should just make `display_name`, `display_coursenumber`, and `display_organization` keyword arguments in `__init__`, since those are the only fields that this summary is concerned with.